src/Scripts/FSScriptLoader.spec.ts

```diff
@@ -0,0 +1,80 @@
+import 'reflect-metadata';
+import { DefaultVirtualScripts } from './DefaultVirtualScripts';
+import { FSScriptLoader } from './FSScriptLoader';
+import { VirtualScriptType } from './VirtualScript';
+import { logger } from '../Utils';
+import {
+  anyFunction,
+  anyString,
+  anything,
+  instance,
+  mock,
+  spy,
+  verify,
+  when
+} from 'ts-mockito';
+import { expect, use } from 'chai';
+import promisified from 'chai-as-promised';
+import fs from 'fs';
+
+use(promisified);
+
+describe('FSScriptLoader', () => {
+  const createScriptLoader = () => {
+    const mockedVirtualScripts = mock(DefaultVirtualScripts);
+    const virtualScripts = instance(mockedVirtualScripts);
+    const scriptLoader = new FSScriptLoader(virtualScripts);
+
+    return { virtualScripts, mockedVirtualScripts, scriptLoader };
```

Comment:
`virtualScripts` is not used in test cases below yet it is returned from this setup function which is not a problem but most likely a refactoring leftover or something? IDK but I'd rather remove it from the resulting object.